test/ibmq/test_ibmq_job.py

```diff
@@ -87,7 +87,6 @@ def test_run_simulator(self, provider):
     def test_run_device(self, provider):
         """Test running in a real device."""
         backend = least_busy(provider.backends(simulator=False))
```

Comment:
Can you do a round of checking the instances where `least_busy()` is used? I'm not sure if this test was left out of the `@required_backend`ification on purpose (based on the PR description), and could be the case for others.